bepasty_cli/cli.py

```diff
@@ -55,12 +76,17 @@
     '--insecure',
     help='Disable SSL certificate validation',
     is_flag=True)
-def main(token, filename, fname, url, ftype, list_pastes, insecure):
+def main(token, filename, fname, url, ftype, list_pastes, insecure, max_lifetime):
     url = url.rstrip("/")
+    max_lifetime = (max_lifetime[0], LIFETIME_MAPPING[max_lifetime[1]])
+    if max_lifetime[0] < 1:
+        print('Invalid "max-lifetime" value "{}". Setting max-lifetime to "forever".'.format(max_lifetime[0]))
+        max_lifetime = (1, 'FOREVER')
```

Comment:
that's a bit unusual - usually a tool would rather fail if it gets invalid parameters.